### PR TITLE
1505 POS Sample Signoff

### PR DESCRIPTION
**Pull request by djonik**
-small UI fix to CashDrawer js app
-removing lingering "DefaultSize" tags
-renaming "Cashdrawer" directory to "CashDrawer"
-fixing links in readme files

### CODE REVIEW

File: CashDrawer/readme.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ To obtain information about Microsoft Visual Studio 2015 and the tools for devel
 
 ### Samples
 
-[Cash Drawer sample](https://github.com/Microsoft/Windows-universal-samples/tree/master/cash-drawer)
+[Cash Drawer sample](https://github.com/Microsoft/Windows-universal-samples/tree/master/cashdrawer)
```
Comment (oldnewthing) [inline CashDrawer/readme.md]: CashDrawer - URls are case-sensitive.

File: MagneticStripeReader/readme.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ To obtain information about Microsoft Visual Studio 2015 and the tools for devel
 
 ### Samples
 
-[Magnetic Stripe Reader sample](https://github.com/Microsoft/Windows-universal-samples/tree/master/magnetic-stripe-reader)
+[Magnetic Stripe Reader sample](https://github.com/Microsoft/Windows-universal-samples/tree/master/magneticstripereader)
```
Comment (oldnewthing) [inline MagneticStripeReader/readme.md]: MagneticStripeReader - URLs are case-sensitive on github. Same for PosPrinter.

File: BarcodeScanner/readme.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ To obtain information about Microsoft Visual Studio 2015 and the tools for devel
 
 ### Samples
 
-[Barcode Scanner sample](https://github.com/Microsoft/Windows-universal-samples/tree/master/barcode-scanner)
+[Barcode Scanner sample](https://github.com/Microsoft/Windows-universal-samples/tree/master/barcodescanner)
```
Comment (oldnewthing) [inline BarcodeScanner/readme.md]: URLs are case-sensitive on github. Should be .../master/BarCodeScanner

File: CashDrawer/readme.md (hunk 1 above)
Comment (djonik) [inline CashDrawer/readme.md]: Thanks for the catch!

**Pull request merged**